api/models/wishlistDao.js

```diff
@@ -0,0 +1,33 @@
+const dataSource = require("./dataSource");
+
+const listWishlist = async (userId) => {
```

Comment:
```suggestion
const getWishlist = async (userId) => {
```
각각의 layer들 모두 `listWishlist` 보다는 CRUD와 연관된 액션(create, get, update, delete) + 목적 자원의 조합으로 REST 엔드포인트 설계하듯이 함수명들도 일관되며 직관적이게 작성할 수 있습니다.